### PR TITLE
Fix test and add elasticsearch to travis

### PR DESCRIPTION
**Pull request by pkrawat1**
## Why?

- Travis tests are failing as elasticsearch is not installed.

## This change addresses the need by:

- adds elasticsearch installer.
- fixes test cases.
- fixes product review test.

## Checklist

- [ ] I have read [CONTRIBUTING.md][contributing].
- [ ] My code follows the [style guidelines][contributing] of this project.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My code does not generate any (new) [`credo`][credo] and compile-time warnings.
- [ ] I have updated the documentation wherever necessary.
- [ ] I have added tests to cover my changes.

[contributing]: https://github.com/aviabird/snitch/CONTRIBUTING.md
[credo]: https://github.com/rrrene/credo

### CODE REVIEW

File: apps/snitch_core/config/test.exs [hunk 1]
```diff
@@ -21,3 +21,19 @@ config :logger, level: :info
 config :argon2_elixir,
   t_cost: 1,
   m_cost: 8
+
+config :snitch_core, Snitch.Tools.ElasticsearchCluster,
+  url: "http://localhost:9200",
```
Comment (SagarKarwande) [inline apps/snitch_core/config/test.exs]: Can we have this Elasticsearch config in `config.exs`.
We have `.env` for `test` also, so we can fetch the config values from environment.

Comment (ashish173) [reply apps/snitch_core/config/test.exs]: hmm, @SagarKarwande can you please mention the pros and cons as well. Thanks

Comment (ashish173) [reply apps/snitch_core/config/test.exs]: hmm, makes sense to me. @pkrawat1 what do you think?

Comment (pkrawat1) [reply apps/snitch_core/config/test.exs]: I'm pushing it to config. I was facing issue while creating an index for testing.

**Pull request merged**